nilearn/plotting/img_plotting.py

```diff
@@ -805,8 +806,25 @@ def plot_prob_atlas(maps_img, anat_img=MNI152TEMPLATE, view_type='auto',
             display.add_contours(map_img, levels=[thr],
                                  linewidths=linewidths,
                                  colors=[color], filled=filled,
-                                 alpha=alpha, linestyles='solid')
-
+                                 alpha=alpha, linestyles='solid',)
+    if colorbar:
+        our_cmap = cmap
+        display._colorbar = True
+        display._show_colorbar(matplotlib.colors.LinearSegmentedColormap.
+                               from_list('segmented colors',
+                                         color_list, n_maps + 1),
+                               matplotlib.colors.Normalize(1, n_maps + 1))
+        tick_locator = matplotlib.ticker.MaxNLocator(nbins=10)
+        display.locator = tick_locator
+        display._cbar.update_ticks()
+        display._cbar.set_ticks(np.round(np.linspace(1, n_maps,
+                                                     min(n_maps, 10))) + .5)
+        display._cbar.set_ticklabels(np.round(np.linspace(1, n_maps,
+                                                 min(n_maps, 10))).astype('int'))
+        left, bottom, width, height = display._colorbar_ax.get_position().bounds
+        display._colorbar_ax.set_position([left, bottom, width, height * 0.95])
+        display._colorbar_ax.annotate('Map #', xy=(1, 1.03), ha='right', va='bottom',
```

Comment:
Would it be possible to support user-supplied labels (sorry if my question is silly, I don't know this code well) ?